test/models/Reads/PensionSchemeAdministratorReadsSpec.scala

```diff
@@ -96,6 +96,20 @@ class PensionSchemeAdministratorReadsSpec extends WordSpec with MustMatchers wit
         result.numberOfDirectorOrPartners mustBe None
       }
 
+      "We have a flag for isMoreThanTenDirectorsOrPartnersChanged" in {
+        val psaWithUpdatedMoreThan10Directors = input + ("isMoreThanTenDirectorsOrPartnersChanged" -> JsBoolean(true)) + ("moreThanTenDirectors" -> JsBoolean(true))
+        val result = psaWithUpdatedMoreThan10Directors.as[PensionSchemeAdministrator](PensionSchemeAdministrator.apiReads)
+
+        result.numberOfDirectorOrPartners.value.isChanged.value mustBe true
+      }
+
+      "We have a flag for areDirectorsOrPartnersChanged" in {
+        val psaWithDirectorsOrPartnersUpdated = input + ("areDirectorsOrPartnersChanged" -> JsBoolean(true))
+        val result = psaWithDirectorsOrPartnersUpdated.as[PensionSchemeAdministrator](PensionSchemeAdministrator.apiReads)
+
+        result.changeOfDirectorOrPartnerDetails.value mustBe true
```

Comment:
Same as above, i.e. say result.changeOfDirectorOrPartnerDetails mustBe Some(true) and add a test for where the flag isn't present.